src/bid/encoding.rs

```diff
@@ -7,13 +7,14 @@
 //! Encoding module for Bid structure.
 //! See: <https://hackmd.io/@7dpNYqjKQGeYC7wMlPxHtQ/BkfS78Y9L>
 
+#[cfg(feature = "canon")]
+use alloc::{vec, vec::Vec};
+#[cfg(feature = "canon")]
+use dusk_plonk::{constraint_system::ecc::Point as PlonkPoint, prelude::*};
+
```

Comment:
Nit: you can use one single `use` and one single `cfg`, but if you prefer kept them split is fine as well.